lunavl/sdk/image_utils/image.py

```diff
@@ -167,13 +167,21 @@ def __init__(
             error = self.coreImage.loadFromMemory(body, len(body), imgFormat)
             if error.isError:
                 raise LunaSDKException(LunaVLError.fromSDKError(error))
+        elif isinstance(body, np.ndarray):
+            arr = body.__array_interface__
```

Comment:
is there another way to do this without accessing to private field?

Reply:
fexed